src/storybook.test.ts

```diff
@@ -37,18 +37,20 @@ describe("storybook", () => {
     test: ({ story, context, stories2snapsConverter, done }) => {
       const snapshotFileName =
         stories2snapsConverter.getSnapshotFileName(context);
+
       // eslint-disable-next-line testing-library/render-result-naming-convention
-      const component = story.render();
-      const { container } = render(component);
-      // Mount components asynchronously to allow for initial state to be set
+      const jsx = story.render();
+      const { unmount, container } = render(jsx);
```

Comment:
What was the motivation behind this?

Reply:
honestly I just added [`unmount`](https://testing-library.com/docs/react-testing-library/api/#unmount) to guarantee that the component is removed, but I think it still works without it